src/EscargotInfo.h.in

```diff
@@ -24,4 +24,8 @@
 #define ESCARGOT_VERSION "@ESCARGOT_BUILD_VERSION@"
 #endif
 
+#ifndef ESCARGOT_BUILD_DATE
+#define ESCARGOT_BUILD_DATE __DATE__
```

Comment:
`ESCARGOT_BUILD_DATE` is always defined. Is this really necessary?
If so, how about adding this macro into `Escargot.h` file because `EscargotInfo.h.in` file is used for cmake build configuration only.